package.json

```diff
@@ -11,50 +11,50 @@
     "vanilla/*"
   ],
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "lerna": "lerna",
-    "clean": "lerna run --parallel clean",
-    "build": "lerna run --parallel build",
-    "lint": "lerna run --parallel lint:es",
-    "lint:fix": "lerna run --parallel lint:es:fix",
-    "tsc": "lerna run --parallel tsc",
-    "test": "lerna run --parallel test",
+    "clean": "npm run clean --workspaces --if-present",
```

Comment:
I seems I always see a `clean` script in every repo I look at, but I've always wondered why NPM doesn't offer this out of the box. There must be a good reason? This is probably one of the few exceptions where I'd say that the root should maintain this functionality for workspaces  because the root is where the `npm i` happens. This can also be replaced with something like `npx remove-node-modules` or `rm -rf node_modules && npm exec --workspaces -- rm -rf node_modules`.